src/console/dlt-receive.c

```diff
@@ -506,11 +514,30 @@ int main(int argc, char *argv[])
 
     if (dltclient.mode == DLT_CLIENT_MODE_TCP || dltclient.mode == DLT_CLIENT_MODE_UDP_MULTICAST) {
         dltclient.port = dltdata.port;
-        for (index = optind; index < argc; index++)
-            if (dlt_client_set_server_ip(&dltclient, argv[index]) == -1) {
+
+        unsigned int servIPLength = 1; // Counting the terminating 0 byte
+        for (index = optind; index < argc; index++) {
+            servIPLength += strlen(argv[index]);
+            if (index > optind)
+            {
+                servIPLength++; // For the comma delimiter
+            }
+        }
+        if (servIPLength > 0) {
```

Comment:
`unsigned int servIPLength = 1; // Counting the terminating 0 byte`
so I believe this check is always true.
Please correct this.